### DISCUSSION TITLE
How to remove default browser?

### DISCUSSION THREAD

**Discussion by coderhasib** [elementary/browser#109] [Q&A]
Hello there,

I need to know what is the name of default web browser in elementary 7 and how to remove it? Thanks.

**Comment by danirabbit** [comment]
The default web browser in elementary OS is GNOME Web.

You can uninstall apps by either:
1. Opening the applications menu and finding the app
2. Secondary-clicking on the app and selecting "Uninstall"

or
1. Open AppCenter and clicking the "Updates and installed apps" icon in the top right corner
2. Select the app from the list of installed apps
3. Click the "Uninstall" button in the top right
